specs/data_structures.md

```diff
@@ -475,7 +480,91 @@ enum VoteType : uint8_t {
 
 # State
 
-TODO validator set repr
+| name              | type                      | description                  |
+| ----------------- | ------------------------- | ---------------------------- |
+| `accountTrieRoot` | [HashDigest](#hashdigest) | Merkle root of account trie. |
```

Comment:
Can't we simply call this accountRoot? And remove the reference to trie below too?

Reply:
Fixed in e8cc2d3.